config/local_ssl/localhost.crt

```diff
@@ -0,0 +1,28 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
this file should be removed -- it got moved to config/certs

Reply:
Done in 4ee6b23.